### PR TITLE
Fix blank feed

### PR DESCRIPTION
**Pull request by gaearon**
The issue was the feed tuner marking all feed items as stale because we've "seen" them. We've "seen" the initial feed fetch was immediately superseded by a second one. That happened because the pinned feeds (which are part of the parameters) were not immediately available, so loading them caused the query to refetch. The fix is to block on the prefs query.

### CODE REVIEW

File: src/view/screens/Home.tsx [hunk 1]
```diff
@@ -7,44 +8,52 @@ import {FollowingEmptyState} from 'view/com/posts/FollowingEmptyState'
...
   const setDrawerSwipeDisabled = useSetDrawerSwipeDisabled()
-  const pagerRef = React.useRef<PagerRef>(null)
   const [selectedPage, setSelectedPage] = React.useState(0)
-  const [customFeeds, setCustomFeeds] = React.useState<FeedDescriptor[]>([])
-  const {data: preferences} = usePreferencesQuery()
-
-  React.useEffect(() => {
-    if (!preferences?.feeds?.pinned) return
 
+  const customFeeds = React.useMemo(() => {
```
Comment (gaearon) [inline src/view/screens/Home.tsx:43]: This is a driveby change, it seems okay? Didn't see a reason for this to be an effect

**Review by pfrazee — APPROVED** [review]
Great
